src/protocol/configuration.jl

```diff
@@ -234,11 +234,20 @@ mutable struct DidChangeConfiguration
     settings::Any
 end
 
-mutable struct ConfigurationItem
-    scopeUri::Union{Nothing,String}    
+@json_read mutable struct ConfigurationItem
+    scopeUri::Union{Nothing,String}
     section::Union{Nothing,String}
 end
 
+function JSON.lower(a::ConfigurationItem)
+    d = Dict{String, Any}()
+
+    a.scopeUri!==nothing && d["scopeUri"] = a.scopeUri
```

Comment:
Have to enclose the assignment in parentheses `&&` or `||` for this to be valid syntax:

```julia
julia> d = Dict()
Dict{Any,Any} with 0 entries

julia> true && d[1] = 5
ERROR: syntax: invalid assignment location "true && d[1]"
Stacktrace:
 [1] top-level scope at REPL[5]:1
 [2] run_backend(::REPL.REPLBackend) at /home/adam/.julia/packages/Revise/439di/src/Revise.jl:975
 [3] (::getfield(Revise, Symbol("##80#82")){REPL.REPLBackend})() at ./task.jl:268

julia> true && (d[1] = 5)
5
```

Reply:
Oh man, how embarrassing :) New push that hopefully fixes that!